tasks/system_probe.py

```diff
@@ -591,10 +596,19 @@ def kitchen_prepare(ctx, windows=is_windows, kernel_release=None):
             if os.path.isdir(extra_path):
                 shutil.copytree(extra_path, os.path.join(target_path, extra))
 
-    if os.path.exists("/opt/datadog-agent/embedded/bin/clang-bpf"):
-        shutil.copy("/opt/datadog-agent/embedded/bin/clang-bpf", os.path.join(KITCHEN_ARTIFACT_DIR, ".."))
-    if os.path.exists("/opt/datadog-agent/embedded/bin/llc-bpf"):
-        shutil.copy("/opt/datadog-agent/embedded/bin/llc-bpf", os.path.join(KITCHEN_ARTIFACT_DIR, ".."))
+    gopath = os.getenv("GOPATH")
+    copy_files = [
+        "/opt/datadog-agent/embedded/bin/clang-bpf",
+        "/opt/datadog-agent/embedded/bin/llc-bpf",
+        f"{gopath}/bin/gotestsum",
+    ]
+
+    files_dir = os.path.join(KITCHEN_ARTIFACT_DIR, "..")
+    for cf in copy_files:
+        if os.path.exists(cf):
+            shutil.copy(cf, files_dir)
+
+    ctx.run(f"go build -o {files_dir}/test2json -ldflags=\"-s -w\" cmd/test2json", env={"CGO_ENABLED": "0"})
```

Comment:
Is `cmd/test2json` not included in the PR?

Reply:
It is part of the Go distribution, so always available.

Reply:
I thought it was doing the opposite :-)